README.md

```diff
@@ -66,3 +66,118 @@ $ wget https://github.com/apple/swift/raw/master/utils/gyb
 $ wget https://github.com/apple/swift/raw/master/utils/gyb.py
 $ chmod +x gyb
 ```
+
+# Efficient way to calculate hash of a large file
+
+To calculate hash of a large file use `DispatchIO`.
+
+In the example below `DispatchIO` reads a file by chunks and process every chunk by calling update method of `SHA256` class.
+
+The default chunk size is set to 128 MB to limit maximum memory usage.
+
+![Screenshot](./scr.png)
+
+```swift
+final class FileHash {
+    
+    private let queue = DispatchQueue(
```

Comment:
`queue` cannot be set externally

Reply:
What do you mean?

Reply:
I've added ability to set work queue in `init`.